src/component/Calendar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import { Calendar, dayjsLocalizer } from "react-big-calendar";
+import dayjs from "dayjs";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import { apiPath } from "../api";
+
+function MyCalendar() {
+  const localizer = dayjsLocalizer(dayjs);
+  const [events, setEvents] = useState([]);
+
+  const fetchMembers = async (teamId) => {
+    try {
+      const responseMembers = await fetch(apiPath(`/members`));
+      const membersData = await responseMembers.json();
+
+      const teamMembers = membersData.filter(
+        (member) => member.team_id === teamId
+      );
+
+      return teamMembers;
+    } catch (error) {
+      throw new Error("Error fetching members:", error);
+    }
+  };
+
+  useEffect(() => {
+    const fetchEvents = async () => {
+      try {
+        const teamCode = window.location.pathname.split("/").pop();
+        const responseTeams = await fetch(apiPath(`/teams`));
+        const teamsData = await responseTeams.json();
+        const team = teamsData.find((team) => team.team_code === teamCode);
```

Comment:
Here, again, I am not sure if I understand correctly, so take my comment with a grain of salt :D 🧂 

a lot of client side filtering is generally never a good idea. Imagine your app scales and your teams array from the db is 600000 teams. 😅 
It also looks like you already have the endpoints (router.get("/:id" or alike) for it, so why not ditch this filtering and just fetch  only what you need right away? 
Exact same principle goes for /members.